config/locales/en.yml

```diff
@@ -131,6 +131,27 @@ en:
         other: Issued %{count} times
         zero: Never issued
       no_certificates: You haven't created any certificates yet!
+    CourseExport__EditDrawer:
```

Comment:
@prashanth726 This key is unused now `CourseExport__EditDrawer` and needs cleanup